### ISSUE TITLE
Add Blog # on RCOS Project Blog and Future Blogs

### ISSUE THREAD

**Issue by CorneliusHsiao** [ZiniuYu/GoogleCardboardVR-AR#6]
Can we add number of blogs on rcos.io so all blogs will be listed from 1 (or 0, if we’re computer scientists)? And to write Blog after every group meeting, can we swich one by one? If you have any accomplishment, please also add a note on Blog site.

**Comment by HarryXu12138** [comment]
What do you mean by "all blogs will be listed from 1?

**Comment by CorneliusHsiao** [comment]
It should be “numbered”. THX

**Comment by HarryXu12138** [comment]
I don't think we should number those blogs since that's not necessary. But about that switch one by one I agree about that. I have some confusion about that blog. What should be post in that blog? For example if we need to post the things we learnt, we should post a blog about the git session.

**Comment by CorneliusHsiao** [comment]
As far as I know, we should write blogs if we: 1) have a meeting, 2) achieve great accomplishment, 3) meet w/ someone and summarize what we learn, 4) make significant decisions, 5) make change to milestones or goals, etc.

**Comment by ziyangji** [comment]
Is the blog use for record our process？

**Comment by CorneliusHsiao** [comment]
Yes, in some way.

**Comment by HarryXu12138** [comment]
Ok, I will write a blog about that git session.
